en/modules/openiddict.md

```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
```

Comment:
```suggestion
This module provides integration and management functionality for the OpenIddict library;
```

Also, give link for who doesn't know it.

Reply:
The next line has the link, shall we add link anyway?

https://github.com/abpio/abp-commercial-docs/blob/c211203d84f67da96907debc4bc117df7285064a/en/modules/openiddict.md?plain=1#L5